include/aikido/robot/ConcreteManipulator.hpp

```diff
@@ -42,6 +42,13 @@ class ConcreteManipulator : public Manipulator
       const dart::dynamics::MetaSkeletonPtr& metaSkeleton,
       const aikido::trajectory::Trajectory* path) override;
 
+  // Documentation inherited.
+  virtual std::unique_ptr<aikido::trajectory::Spline> retimePathWithKunzTimer(
```

Comment:
We should create an issue stating this is temporary and should be replaced with a commoon Postprocessor class that Hauser, Kunz derive from that can be passed into these functions.